### PR TITLE
Bugfix/only draw with pointer

### PR DESCRIPTION
**Pull request by BrandonE**

### CODE REVIEW

File: src/view.js [hunk 1]
```diff
@@ -272,6 +268,10 @@ function bindEvents(ctrl, el, context) {
 }
 
 function renderBoard(ctrl) {
+  ctrl.pointerSelected = function pointerSelected() {
+    return !ctrl.sparePieceSelected || ctrl.sparePieceSelected === 'pointer';
+  };
```
Comment (ornicar) [inline src/view.js]: I don't understand why this function is declared here. Why not in ctrl.js instead? Sounds quite natural to me.

**Comment by BrandonE** [comment]
Chessground has no awareness of what the currently selected tool is in lila, right? I'm not sure how to resolve this conflict without making that information available to chessground.

**Comment by ornicar** [comment]
maybe just `if (rightClick && sel !== 'pointer') chessground.state.drawable.current = undefined`

**Comment by BrandonE** [comment]
Fixed in https://github.com/ornicar/lila/pull/2737

**Pull request closed without merging**